test/jdk/javax/net/ssl/SSLEngine/Basics.java

```diff
@@ -26,19 +26,23 @@
  * @bug 4495742
  * @summary Add non-blocking SSL/TLS functionality, usable with any
  *      I/O abstraction
- *
  * This is intended to test many of the basic API calls to the SSLEngine
  * interface.  This doesn't really exercise much of the SSL code.
  *
+ * @library /test/lib
  * @author Brad Wetmore
+ * @run main Basics
  */
 
 import java.security.*;
 import java.io.*;
 import java.nio.*;
+import java.util.Arrays;
 import javax.net.ssl.*;
 import javax.net.ssl.SSLEngineResult.*;
 
+import jdk.test.lib.security.SecurityUtils;
+
 public class Basics {
 
     private static String pathToStores = "../etc";
```

Comment:
optional - this is an old code but since you are refactoring this test, can you please declare these as constants as well.